backend/monitoring/core_integration.py

```
@@ -0,0 +1,215 @@
+            asyncio.create_task(self.integration_monitor.start_monitoring())
+            
+            self.initialized = True
+            logger.info("✅ Monitoring core integration initialized")
+            
+        except Exception as e:
+            logger.error(f"❌ Failed to initialize monitoring: {e}")
+            raise
+            
+    def create_middleware(self):
+        """Create FastAPI middleware for monitoring"""
+        from fastapi import Request, Response
```

Ruff (0.12.2)

`fastapi.Response` imported but unused

Remove unused import: `fastapi.Response`

(F401)